.github/workflows/ci.yml

```diff
@@ -36,5 +36,7 @@ jobs:
           name: coverage
           path: .test/coverage**
           if-no-files-found: error
+      - name: gofmt
+        run: find -name '*.go' -type f -exec ./.go-env.sh gofmt -l -s -w '{}' +
```

Comment:
Do you not need to supply a path to `find`?

Reply:
It defaults to the current working directory :+1:

Reply:
Must be a GNU find enhancement. BSD find errors.

Reply:
Even BusyBox is happy with it! :sweat_smile: 

```console
$ docker run --rm busybox find -maxdepth 1 -name etc
./etc
```

Reply:
Should I read that as a plus or a minus?
/me ducks